configs/api/api.go

```diff
@@ -50,10 +50,18 @@ func (a *API) RegisterRoutes(r *mux.Router) {
 		handler            http.HandlerFunc
 	}{
 		{"root", "GET", "/", a.admin},
+		// Deprecated public APIs
 		{"get_config", "GET", "/api/configs/org/cortex", a.getConfig},
 		{"set_config", "POST", "/api/configs/org/cortex", a.setConfig},
+		// Dedicated APIs for updating rules config. In future, these *must*
```

Comment:
Changed, but see https://jakubmarian.com/in-the-future-vs-in-future-in-british-and-american-english/